sdks/python/apache_beam/runners/worker/logger.py

```diff
@@ -49,7 +48,7 @@ def get_data(self):
 per_thread_worker_data = _PerThreadWorkerData()
 
 
-class PerThreadLoggingContext(LoggingContext):
+class PerThreadLoggingContext(object):
```

Comment:
Are we able to get rid of this class entirely?  It looks like you removed the only usage in operations.py.